XenAdmin/Controls/ListViewEx.cs

```diff
@@ -43,5 +43,31 @@ public ListViewEx()
             // Magically eliminates flicker when the control is resized
             this.SetStyle(ControlStyles.OptimizedDoubleBuffer, true);
         }
+
+        protected override void OnColumnWidthChanging(ColumnWidthChangingEventArgs e)
+        {
+            base.OnColumnWidthChanging(e);
+
+            ColumnHeaderEx columnex = Columns[e.ColumnIndex] as ColumnHeaderEx;

```

Comment:
I'm wondering if we can handle ColumnHeaderEx in a general way. 
Now the code is to check if the column is of type ColumnHeaderEx, and then do something if it is. I think it's not very good to handle a concrete type here. 
I haven't figured out a better idea. Any way, this is not a request to modify anything. I just want to discuss if it's possible.